.github/workflows/ci.yml

```diff
@@ -80,13 +78,19 @@ jobs:
           version: current-test
           push: true
           distball: ${{ steps.build-zeebe.outputs.distball }}
-      - name: Prepare Testcontainers Cloud agent
-        if: env.TC_CLOUD_TOKEN != ''
-        run: |
-          curl -L -o agent https://app.testcontainers.cloud/download/testcontainers-cloud-agent_linux_x86-64
-          chmod +x agent
-          ./agent --private-registry-url=http://localhost:5000 '--private-registry-allowed-image-name-globs=*,*/*' --terminate > .testcontainers-agent.log 2>&1 &
-          ./agent wait
+      - name: Setup TCC
+        if: matrix.tcc-enabled == 'true'
```

Comment:
:thought_balloon: I was disappointed to see you can't just write `if: matrix.tcc-enabled` :( But maybe I'm missing something